app/elements/youneeq-tracker/youneeq-tracker.js

```diff
@@ -96,11 +98,12 @@ class youneeqTracker {
 
   sendPageHit(event) {
     console.info('<\youneeq-tracker\> page hit event received');
-    if(event.detail){
+    if(typeof event.detail !== 'undefined' && typeof event.detail.content !== 'undefined'){
       let content = event.detail.content;
-      this._pageHitChanged(content);
+      this.set('content', content);
     } else {
-      this._pageHitChanged(event);
+      // Set content to the event to ensure that stale data doesn't sit within content.
+      this.set('content', event);
     }
   }
```

Comment:
I also modified this method as it should not be calling the observer handler directly, but allowing Polymer to run through its lifecycle. This will ensure that this.async functions fire properly.